documentation/book/assembly-setting-up-tracing-mirror-maker-connect.adoc

```diff
@@ -0,0 +1,13 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-tracing.adoc
+[id='assembly-setting-up-tracing-mirror-maker-connect-{context}']
+= Setting up tracing for Mirror Maker and Kafka Connect
+
+Distributed tracing is supported for Mirror Maker and Kafka Connect (including Kafka Connect with Source2Image support). 
+
+For Mirror Maker, messages are traced from the source cluster to the target cluster; the trace data visualize messages entering and leaving the Mirror Maker component.
```

Comment:
data visualizes? Not sure what that means. The trace data is used to visualize...?

Reply:
Indeed, this doesn't make much sense without referring to the Jaeger user interface. I've changed to: "...the trace data records messages entering and leaving the Mirror Maker component."